server/views/pages/goal/create/index.njk

```diff
@@ -27,14 +36,20 @@
         <div class="govuk-form-group">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
+          {% set titleLabelHtml %}
```

Comment:
I took the opportunity to update the page content from Jo's content docs ... though I think the slack conversation about it is making me wish I hadn't have!